airbyte-integrations/bases/base-normalization/normalization/transform_catalog/catalog_processor.py

```diff
@@ -124,6 +140,10 @@ def build_stream_processor(
                 destination_type=destination_type,
                 raw_schema=raw_schema_name,
                 schema=schema_name,
+                source_sync_mode=source_sync_mode,
```

Comment:
i'm having trouble understanding where source_sync_mode is used after it is passed in there. am i just missing it or is it a future thing?

Reply:
yes you're right it's not currently used, this Is just registering what the source sync was and can be used in the future on the normalization side